kontrakter/politi/varetekt/1.0/begjaeringVaretekt.schema.json

```diff
@@ -1,45 +1,43 @@
 {
-  "$schema": "http://json-schema.org/schema#",
-  "description": "Skjema for begjæring av varetektsmeldinger fra påtale til tingretten",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://politiet.no/politi/fengsling/1.0/begjaeringVaretekt",
+  "description": "Skjema for begjæring av varetektsmeldinger fra påtale til tingretten, JSON Schema versjon 2020-12",
 
   "type": "object",
   "properties": {
-	  "begjaeringVaretekt": {"$ref": "#/$defs/begjaeringVaretekt"}
+    "melding": {"type": "string"},
```

Comment:
`melding` her og `aktoerer` under er mystiske for meg. Hva er disse for?

Reply:
aktoerer henger igjen ennå, ikke ferdig.
melding er navn på  meldingstype og skal være konstant.

Reply:
forsvarsadvokat henger på siktede, slettet aktører.